bentoml/_internal/io_descriptors/json.py

```diff
@@ -38,20 +37,23 @@ def default(self, o: _SerializableObj) -> t.Any:
         if dataclasses.is_dataclass(o):
             return dataclasses.asdict(o)
 
-        if isinstance(o, np.generic):
-            return o.item()
+        if "numpy" in sys.modules:
```

Comment:
@larme @aarnphm this ensures that if user is not already using numpy in their service code, it won't get imported at all